ios/PurchasesHybridCommon/PurchasesHybridCommon/RCErrorContainer.m

```diff
@@ -28,8 +28,14 @@ - (instancetype)initWithError:(NSError *)error extraPayload:(NSDictionary *)extr
     }
 
     if (error.userInfo[RCReadableErrorCodeKey]) {
-        dict[@"readableErrorCode"] = error.userInfo[RCReadableErrorCodeKey];
-        dict[@"readable_error_code"] = error.userInfo[RCReadableErrorCodeKey];
+        NSString *readableErrorCode = error.userInfo[RCReadableErrorCodeKey];
+        dict[@"readableErrorCode"] = readableErrorCode;
+        dict[@"readable_error_code"] = readableErrorCode;
```

Comment:
since this is the same as `RCReadableErrorCodeKey` [as declared here](https://github.com/RevenueCat/purchases-ios/blob/d0ac5b9b422207f7c710d04cb77899e4e77d9495/Purchases/Public/RCPurchasesErrorUtils.m#L22), should we be using that instead?